plugins/inputs/lvm/lvm.go

```diff
@@ -33,7 +33,7 @@ func (*LVM) SampleConfig() string {
 	return sampleConfig
 }
 
-func (lvm *LVM) Init() error {
+func (*LVM) Init() error {
 	return nil
 }
```

Comment:
Please remove this function altogether as it is optional anyway.

Reply:
Removed